demos/handwritten_text_recognition_demo/python/README.md

```diff
@@ -77,7 +77,10 @@ Options:
                         Optional. Top k steps in looking up the decoded
                         character, until a designated one is found
   -ob OUTPUT_BLOB, --output_blob OUTPUT_BLOB
-                        Optional. Name of the output layer of the model. Default is 'output'
+                        Optional. Name of the output layer of the model.
```

Comment:
does this message matched with message printed in demo with --help?

Reply:
Yes, the help message was copied here word for word.